configure_me/README.md

```diff
@@ -28,47 +30,60 @@ Let's say, your application needs these parametrs to run:
 * Port - this is mandatory
 * IP address to bind to - defaults to 0.0.0.0
 * Path to TLS certificate - optional, the server will be unsecure if not given
+* RunFast - switch, if present the server will run in fast mode
 
 First create `config_spec.toml` configuration file specifying all the parameters:
 
 ```toml
 [[param]]
+# short param, e.g -p
+abbr = "p"
+# long arg, e.g --port
 name = "port"
 type = "u16"
 optional = false
+
 # This text will be used in the documentation (help etc)
 # It's not mandatory, but your progam will be ugly without it.
 doc = "Port to listen on."
 
 [[param]]
 name = "bind_addr"
-type = "::std::net::Ipv4Addr" # Yes, this works and  you can use your own types implementing Deserialize and ParseArg as well!
+# Yes, this works and  you can use your own T: Deserialize + ParseArg as well!
+# merge from env var. e.g BIND_ADDR
+env_var = true
+type = "::std::net::Ipv4Addr"
 default = "::std::net::Ipv4Addr::new(0, 0, 0, 0)" # Rust expression that creates the value
 doc = "IP address to bind to."
 
 [[param]]
 name = "tls_cert"
-type = "String"
+type = "::std::path::PathBuf"
 doc = "Path to the TLS certificate. The connections will be unsecure if it isn't provided."
 # optional = true is the default, no need to add it here
+# If the type is optional, it will be represented as Option<T>
+# e.g. Option<::std::path::PathBuf> in this case.
+
+[[switch]]
+# switches are parsed as bools, true if present false otherwise
+name = "run_fast"
+doc = "Will make the server run in fast mode"
 ```
 
-Then, create a simple `build.rs` script like:
+Then, create a simple build script:
 
 ```rust
-extern crate configure_me_codegen;
+extern crate configure_me;
 
-fn main() -> Result<(), configure_me_codegen::Error> {
-    configure_me_codegen::build_script_auto()
+fn main() {
+    configure_me::build_script_auto().unwrap_or_exit();
 }
 ```
 
-*Tip: use [`cfg_me`](https://github.com/Kixunil/cfg_me) to generate a man page for your program.*
```

Comment:
Why was this removed? There's no way for people to generate man page without knowing about `cfg_me`.